libherokubuildpack/Cargo.toml

```diff
@@ -34,6 +36,7 @@ termcolor = { version = "1.1.3", optional = true }
 thiserror = { version = "1.0.35", optional = true }
 toml = { version = "0.5.9", optional = true }
 ureq = { version = "2.5.0", optional = true }
+crossbeam = { version = "0.8.2", optional = true }
```

Comment:
The `crossbeam` crate itself is just a re-export of half a dozen other crossbeam crates:
https://github.com/crossbeam-rs/crossbeam#crates

It looks like the `crossbeam` features used in this PR are from the `crossbeam-utils` crate only.

Using that crate standalone seems to be a supported/expected use-case, given the docs:
https://crates.io/crates/crossbeam-utils

Please could we switch to `crossbeam-utils` for now, to reduce the number of dependencies that are downloaded and compiled but never needed? We can always switch back to the `crossbeam` crate if we ever start using several of the `crossbeam-*` crates :-)

Also pedantic nit: The dependencies list was previously alphabetical, now it's not 😆

Reply:
Changed in [49ac7ea](https://github.com/heroku/libcnb.rs/pull/535/commits/49ac7ea4752b57a81d908cef478f230d109834fe)